jet/wordcount-compute-isolation/src/main/java/com/hazelcast/samples/jet/wordcount/WordCountWithComputeIsolation.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.samples.jet.wordcount;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.jet.JetMemberSelector;
+import com.hazelcast.jet.JetService;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sinks;
+import com.hazelcast.jet.pipeline.Sources;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Pattern;
+
+import static com.hazelcast.examples.helper.LicenseUtils.ENTERPRISE_LICENSE_KEY;
+import static com.hazelcast.function.Functions.wholeItem;
+import static com.hazelcast.jet.Traversers.traverseArray;
+import static com.hazelcast.jet.aggregate.AggregateOperations.counting;
+import static java.util.Comparator.comparingLong;
+
+/**
+ * Demonstrates a simple Word Count job in the Pipeline API with member selector.
+ * Inserts the text of The Complete Works of William Shakespeare into a Hazelcast
+ * IMap, then lets Jet count the words in it and write its findings to
+ * another IMap. The example looks at Jet's output and prints the 100 most
+ * frequent words.
```

Comment:
```suggestion
 * Demonstrates a simple Word Count job in the Pipeline API with a member selector.
 * Inserts the text of The Complete Works of William Shakespeare into a Hazelcast
 * IMap, then lets Jet count the words in it and write its findings to
 * another IMap. This execution takes place on selected members instead of all
 * available members. The example looks at Jet's output and prints the 100 most
 * frequent words.
```

I think it would be nice to add a bit more fanfare about the isolation 😄

Reply:
https://github.com/hazelcast/hazelcast-code-samples/pull/632/commits/fd53e0bf500f5f584750d1f74cee6caf0310f976